lib/ansible/module_utils/crypto.py

```diff
@@ -132,7 +132,9 @@ def get_fingerprint_of_bytes(source):
 
 
 def get_fingerprint(path, passphrase=None):
-    """Generate the fingerprint of the public key. """
+    """Generate the fingerprint of the public key. 
```

Comment:
```suggestion
    """Generate the fingerprint of the public key from the private key."""
```
(And remove the next line.)

Reply:
done